pycqed/analysis/pytomo.py

```diff
@@ -0,0 +1,909 @@
+from cmath import *
```

Comment:
More tomo helpers I guess
File would benefit from 
- moving to a tomo subfolder 
- a comment as to the purpose of this file.